src/ng-xform/measure-field/measure-field.component.html

```diff
@@ -14,11 +18,10 @@
         <span class="caret"></span>
       </span>
       <ul class="dropdown-menu dropdown-menu-right" role="menu" *dropdownMenu>
-          <li  *ngFor="let item of availableUnits">
-            <a href="javascript:void(0);" (click)="changeUnit(item)">{{ item }}</a>
-          </li>
-        </ul>
+        <li *ngFor="let item of availableUnits">
+          <a href="javascript:void(0);" (click)="changeUnit(item)">{{ item }}</a>
```

Comment:
Move (click)="changeUnit(item)" to list item <li>and replace reference element <a> by <span> since href is not used.

Reply:
in this way, change cursor style to a pointer will be necessary

Reply:
On components CSS, add following lines.
li {
    cursor: pointer;
}
I would prefer that as the clickable area for a link is only there is text. In list item, is the whole element area... but anyway that is minor.